app.js

```diff
@@ -546,6 +550,7 @@ define(function(require) {
 
 			_.each(reviewData, function(userData) {
 				var newData = self.formatUserData(userData, data.customizations ? data.customizations : {});
+				newData.addUsersToMainDirectory = addUsersToMainDirectory;
```

Comment:
Mind moving this into the dedicated formatting method?

Obviously, the second argument has to be updated, so maybe pass the whole `data` variable?

Reply:
Added a custom options argument to `formatUserData`